packages/mixpanel/src/msw/handlers.ts

```diff
@@ -0,0 +1,90 @@
+import { delay, http, HttpResponse, type HttpHandler } from "msw";
+import { MixpanelTrackingResponse, type MixpanelTrackingRequest } from "../mocks/index.ts";
+import { MswMixpanelApiUrls } from "./urls.ts";
+
+/**
+ * Default delay for simulating network latency (in milliseconds)
+ */
+const DEFAULT_DELAY = 100;
+
+/**
+ * MSW handlers for Mixpanel API endpoints
+ */
+export const MixpanelApiHandlers = {
```

Comment:
Do we really need anything more than an handler returning a `200` ?

Reply:
In the sample of this repo, we do have an MSW handler replacing the tracking service proxy, meaning that it actually forward the request to mixpanel.

I don't think we do need this for the host app, do we?